PRODUCTIONISATION.md

```diff
@@ -107,32 +106,38 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 * [ ] 8.1 Sub-organisation created under parent organisation for each customer
 * [ ] 8.2 Monitoring agent be installed on all production infrastructure
 * [ ] 8.3 Monitoring agent could be installed on non-production infrastructure if necessary
-* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008]
+* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008] (_this doesn't apply to serverless_)
```

Comment:
I think section 8 needs a re-think, as:

We have since learned that we need monitoring on serverless deployments (madetech.com downtime due to database connectivity issues, Monitoring + scaling of Fargate instances).
While I accept that "conventional" infrastructure monitoring doesn't work, we still need something, and the exclusion set above has caused us to drop the ball a few times.

>Monitoring Agent

Agents are not the only way to monitor infrastructure. Wording needs generalising.

Reply:
I think this can be reviewed later in order to prioritise rolling out. Created a Trello card and marking as resolved for now.